sdk/core/azure-core/inc/azure/core/context.hpp

```diff
@@ -391,12 +391,17 @@ namespace Azure { namespace Core {
           = ContextSharedState::ToMsecSinceEpoch(time_point::min());
     }
 
+    /**
+     * @brief Checks if the context was canceled.
```

Comment:
I think doxygen comments for C++ should describe verb/action instead of a function, so I would put it as "Check if the context is canceled" - because it is what you see when you invoke intellisense. You can see examples throughout this file and other places, although I don't think there is formal agreement.

I would fill out the section completely and not leave that to future doc reviews, by adding "`@return "`true` if this context is canceled, `false` otherwise.`"